From d377b4503041e20e2034dc5d5b7dbfce7e031f33 Mon Sep 17 00:00:00 2001
From: Bruno Haible <bruno@clisp.org>
Date: Fri, 19 Jul 2019 23:11:48 +0200
Subject: [PATCH 2/9] intl: Make autoconf macro more robust.

Reported by Bernhard Voelker <mail@bernhard-voelker.de>.

* gettext-runtime/m4/intl.m4 (gt_INTL_SUBDIR_CORE): Don't make assumptions about
the version numbers of future Bison versions.
---
 gettext-runtime/m4/intl.m4 | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/gettext-runtime/m4/intl.m4 b/gettext-runtime/m4/intl.m4
index 110766192..f31f274f0 100644
--- a/gettext-runtime/m4/intl.m4
+++ b/gettext-runtime/m4/intl.m4
@@ -1,4 +1,4 @@
-# intl.m4 serial 37 (gettext-0.20)
+# intl.m4 serial 38 (gettext-0.20.2)
 dnl Copyright (C) 1995-2014, 2016-2019 Free Software Foundation, Inc.
 dnl This file is free software; the Free Software Foundation
 dnl gives unlimited permission to copy and/or distribute it,
@@ -269,10 +269,10 @@ changequote(<<,>>)dnl
     ac_prog_version=`$INTLBISON --version 2>&1 | sed -n 's/^.*GNU Bison.* \([0-9]*\.[0-9.]*\).*$/\1/p'`
     case $ac_prog_version in
       '') ac_prog_version="v. ?.??, bad"; ac_verc_fail=yes;;
-      2.[7-9]* | [3-9].*)
+      1.* | 2.[0-6] | 2.[0-6].*)
 changequote([,])dnl
-         ac_prog_version="$ac_prog_version, ok"; ac_verc_fail=no;;
-      *) ac_prog_version="$ac_prog_version, bad"; ac_verc_fail=yes;;
+         ac_prog_version="$ac_prog_version, bad"; ac_verc_fail=yes;;
+      *) ac_prog_version="$ac_prog_version, ok"; ac_verc_fail=no;;
     esac
     AC_MSG_RESULT([$ac_prog_version])
   fi
